docs/src/pages/index.js

```diff
@@ -44,7 +40,7 @@ export async function getStaticProps() {
   return {
     props: {
       branch: process.env?.GIT_BRANCH,
-      cascaraVersion,
+      cascaraVersion: process.env?.npm_package_version,
```

Comment:
Now that we are syncing the docs package with Cascara as far as version in the package.json, we can use the safer npm_package env vars from the package.json without importing it from the other package.